.travis.yml

```diff
@@ -136,7 +136,7 @@ jobs:
         - MB_PYTHON_VERSION=3.8
         - PLAT=aarch64
         - CYTHON_BUILD_DEP="Cython"
-        - DOCKER_TEST_IMAGE=multibuild/focal_{PLAT}
+        - DOCKER_TEST_IMAGE=multibuild/xenial_{PLAT}
```

Comment:
Is the motivation here to test with "oldest supported" image/dependencies where possible, instead of bleeding edge?